bootstrap

```diff
@@ -112,8 +113,10 @@ extract(Binary) ->
     {ok, Contents}.
 
 request(Url) ->
+    HttpOptions = [{relaxed, true} | get_proxy_auth()],
```

Comment:
Sure, I'll fix it.  Also the changes for rebar_utils are missing in this change. I'll add it too. By the way how do I run the tests for rebar? I found this script https://github.com/lafka/dotconfig/blob/master/bin/eunit-module and launch it like this:
./eunit-module rebar_utils_SUITE test/ _build/bootstrap/lib/rebar/ebin but it says:

[rebar3]$ ./eunit-module rebar_utils_SUITE test/ _build/bootstrap/lib/rebar/ebin
> erlc -I include -DTEST -o _build/bootstrap/lib/rebar/ebin test//rebar_utils_SUITE.erl -pa _build/bootstrap/lib/rebar/ebin && \
> erl  -noshell -pa _build/bootstrap/lib/rebar/ebin \ -eval "eunit:test(rebar_utils_SUITE, [verbose])." -run init stop
----------------------------------------------------------------------
======================== EUnit ========================
module 'rebar_utils_SUITE'
  There were no tests to run.